src/stylesheets/components/_search.scss

```diff
@@ -11,69 +11,68 @@ $usa-btn-big-width:     11.6rem;
 
 .usa-search {
   @include clearfix;
-  max-width: none;
+  @include typeset;
   position: relative;
 
+  [role=search] {
+    display: flex;
+  }
+
   [type=submit],
```

Comment:
Should we use `.usa-button` here instead?

Reply:
I think it enforces good markup so don't think so.